integration/test/ca_bundle/ca_bundle_test.go

```diff
@@ -89,16 +79,20 @@ func copyFile(pathIn string, pathOut string) {
 	log.Printf("File : %s copied to : %s", pathIn, pathOut);
 }
 
-func startAgent() {
+func startAgent() string {
 	cmd := exec.Command("bash", "-c", "sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a " +
 		"fetch-config -m ec2 -s -c file:" +
 		configOutputPath)
+	var buff bytes.Buffer
+	cmd.Stderr = &buff
+
 	err := cmd.Run()
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	log.Printf("Agent has started");
+	log.Printf("Agent has started")
+	return buff.String()
```

Comment:
I think you need to return a pointer to the buffer instead of the string. `buff.String` will only give you the contents of the buffer up to the return statement. Then you can dump the buffer in your target check or when you stop the agent.